mscore/musescore.cpp

```diff
@@ -2990,6 +2993,19 @@ void MuseScore::clipboardChanged()
       }
 
 //---------------------------------------------------------
+//   inputMethodLocaleChanged
+//---------------------------------------------------------
+
+void MuseScore::inputMethodLocaleChanged()
+      {
+      QInputMethod* im = QGuiApplication::inputMethod();
```

Comment:
just a note, I'm not testing for (im == NULL) because http://doc.qt.io/qt-5/qguiapplication.html#inputMethod doesn't saying anything about it returning NULL, so I'm assuming it always returns a valid pointer to a QInputMethod object.